test/unittests/mongoBackend/mongoUpdateContextGeo_test.cpp

```diff
@@ -207,6 +207,7 @@ TEST(mongoUpdateContextGeoRequest, newEntityLocAttribute)
     EXPECT_FALSE(a2.hasField("creDate"));
     EXPECT_FALSE(a2.hasField("modDate"));
     EXPECT_STREQ("A1", ent.getObjectField("location").getStringField("attrName"));
+    ASSERT_TRUE(ent.hasField("location"));
```

Comment:
The ASEERT_TRUE statement introduced in this test are a hardening measure for this test: it becomes a segfault in a test error.

NTC